gdcdictionary/schemas/aligned_reads_index.yaml

```diff
@@ -22,18 +22,21 @@ systemProperties:
   - error_type
 
 links:
-  - name: submitted_aligned_reads_files
-    backref: aligned_reads_indexes
-    label: derived_from
-    target_type: submitted_aligned_reads
-    multiplicity: one_to_one
+  - exclusive: false
```

Comment:
what is this for?

Reply:
exclusive means only allowing one of them to present